torch/csrc/jit/passes/python_print.cpp

```diff
@@ -934,11 +934,26 @@ struct PythonPrintPass {
   }
 
   void printOpName(TaggedStringStream& stmt, Symbol kind) {
+    const static std::unordered_set<Symbol> autograd_ns_symbols = {
```

Comment:
This feels overly specific to autograd, it can just be a map from symbol to an override for how it is serialized:
```
{aten::backward, "torch.autograd.backward"},
```